cmd/zpool/zpool_main.c

```diff
@@ -4272,13 +4272,17 @@ print_iostat_header(iostat_cbdata_t *cb)
  * by order of magnitude. Uses column_size to add padding.
  */
 static void
-print_stat_color(char *statbuf, unsigned int column_size)
+print_stat_color(const char *statbuf, unsigned int column_size)
 {
 	fputs("  ", stdout);
+	size_t len = strlen(statbuf);
+	while (len < column_size) {
+		fputc(' ', stdout);
+		column_size--;
+	}
 	if (*statbuf == '0') {
 		color_start(ANSI_GRAY);
 		fputc('0', stdout);
-		column_size--;
```

Comment:
How about adding `color_end();` and `return;` here ....
And we can put the whole `for()` loop a tab to the left.

Reply:
Good point. May be we can do these code cleanups later.
Now I also feels that the series of `else if` inside `for` should probably be turned into a `switch` instead for readability.